src/card.css

```diff
@@ -0,0 +1,95 @@
+.product {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  flex-wrap: wrap;
+  align-items: center;
+  width: 200px;
+  padding: 32px 16px 16px;
+  color: #060b35;
+  background-color: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.product__name {
+  margin-bottom: 4px;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+  float: left;
+}
+
+.product__code-article {
+  align-self: flex-start;
+  margin-bottom: 16px;
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+  font-style: normal;
+  color: #616070;
+}
+
+.product__rating {
+  display: flex;
+  align-items: flex-start;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 24px;
+}
+
+.product__reviews {
+  font-size: 10px;
+  font-weight: 400;
+  line-height: 14px;
+}
+
+.product__price {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  margin-bottom: 16px;
+}
+
+.product__price-name {
+  font-size: 12px;
+  font-weight: 400;
+  line-height: 18px;
+  color: #616070;
+}
+
+.product__price-value {
+  display: flex;
+  align-items: flex-end;
+  font-size: 16px;
+  line-height: 18px;
+  font-weight: 700;
+  text-align: right;
+}
+
+.product__button {
+  width: 166px;
```

Comment:
Avoid hardcoding when it is possible
Read about block elements

Reply:
done